Code/Tools/SceneAPI/SceneBuilder/SceneImporter.cpp

```diff
@@ -61,26 +64,58 @@ namespace AZ
                 }
             }
 
+            SceneAPI::SceneImportSettings SceneImporter::GetSceneImportSettings(const AZStd::string& sourceAssetPath) const
+            {
+                // Start with a default set of import settings.
+                SceneAPI::SceneImportSettings importSettings;
+
+                // Try to read in any global settings from the settings registry.
+                if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry)
+                {
+                    settingsRegistry->GetObject(importSettings, AZ::SceneAPI::DataTypes::IImportGroup::SceneImportSettingsRegistryKey);
+                }
+
+                // Try reading in the scene manifest (.assetinfo file), which contains the import settings if they've been
+                // changed from the defaults.
+                Containers::Scene scene("");
```

Comment:
Using `Containers::Scene scene(AZStd::string{})` for a slightly quicker initialization as the AZStd::string default constructor can be used instead of it's `const char*` constructor.

Though the better solution is to update  the `Containers::Scene` class to have a default constructor.

Reply:
Added the default constructor.